shardingsphere-jdbc/shardingsphere-jdbc-core/src/test/java/org/apache/shardingsphere/driver/jdbc/core/statement/ShardingSpherePreparedStatementTest.java

```diff
@@ -430,7 +430,7 @@ public void assertAddOnDuplicateKey() throws SQLException {
             preparedStatement.setString(8, status);
             preparedStatement.setString(9, updatedStatus);
             int result = preparedStatement.executeUpdate();
-            assertThat(result, is(2));
+            assertThat(result, is(4));
```

Comment:
I'm not sure why `H2Database 1.x` is `2`. At least `4` is reasonable according to the processing logic of `H2Database 2.x`, I don't know what I overlooked.